ingest-app/docs/api.md

```diff
@@ -640,6 +640,7 @@ The following update types are supported:
   * Clear all and replace - clear the list and replace with the update value.
   * Find and replace - replace any instance in the list that matches the find value with the update value.
   * Find and update - merge update value into any instance in the list that matches the find value.
+  * Find and update home page url - A special case for Find and update that only applies to datacenter update. 
```

Comment:
Maybe change to: a special case for Find and update that applies specifically to data center home page url update

Reply:
Per our discussion, we will only say "A special case for Find and update." to avoid confusion.  Detailed explanation could be found in the home page.